docs/blog/posts/introduction-to-bpfd.md

```diff
@@ -0,0 +1,605 @@
+---
+date: 2023-07-16
+authors:
+  - anfredette
+  - shaneutt
+---
+
+# Introduction To bpfd
```

Comment:
I guess this is the title right, I think we can make it a bit more catchy 😉 

```suggestion
# bpfd: a novel way to mange eBPF
```
or something along those lines